File: src/Modules/General/Leaderboards.cs
using DEA.Common.Extensions;
using DEA.Database.Models;
using Discord;
using Discord.Commands;
using System.Linq;
using System.Threading.Tasks;

namespace DEA.Modules.General
{
    public partial class General
    {
        [Command("Leaderboards")]
        [Alias("lb", "rankings", "highscores")]
        [Summary("View the richest Drug Traffickers.")]
        public async Task Leaderboards()
        {
            var users = (await _userRepo.AllAsync(y => y.GuildId == Context.Guild.Id)).OrderByDescending(x => x.Cash);
            string description = string.Empty;
            int position = 1;

            if (users.Count() == 0)
            {
                ReplyError("There is nobody on the leaderboards yet.");
            }

            var guildInterface = Context.Guild as IGuild;

            foreach (User dbUser in users)
            {
                var user = await guildInterface.GetUserAsync(dbUser.UserId);
                if (user == null)
                {
                    continue;
                }

                description += $"{position}. {user.Boldify()}: {dbUser.Cash.USD()}\n";
                if (position >= Config.LeaderboardCap)
                {
                    break;
                }

                position++;
            }

            await SendAsync(description, "The Richest Traffickers");
        }
    }
}

File: src/commands/general/leaderboards.js
const db = require('../../database');
const config = require('../../config.json');
const patron = require('patron.js');
const util = require('../../utility');

class Leaderboards extends patron.Command {
  constructor() {
    super({
      name: 'leaderboards',
      aliases: ['lb', 'highscores', 'highscore', 'leaderboard'],
      group: 'general',
      description: 'View the richest Drug Traffickers.'
    });
  }

  async run(msg) {
    const users = await db.userRepo.findMany({ guildId: msg.guild.id });
		
    users.sort((a, b) => b.cash - a.cash);

    let message = '';
    let position = 1;

    for (let i = 0; i < users.length; i++) {
      if (position > config.leaderboardCap) {
        break;
      }

      const user = msg.client.users.get(users[i].userId);

      if (user === undefined) {
        continue;
      }

      message += position++ + '. ' + util.StringUtil.boldify(user.tag) + ': ' + util.NumberUtil.format(users[i].cash) + '\n';
    }

    if (util.StringUtil.isNullOrWhiteSpace(message)) {
      return util.Messenger.replyError(msg.channel, msg.author, 'There is nobody on the leaderboards.');
    }

    return util.Messenger.send(msg.channel, message, 'The Richest Traffickers');
  }
}

module.exports = new Leaderboards();
